test/ccd2rgb.jl

```diff
@@ -0,0 +1,31 @@
+@testset "ccd2rgb" begin
+    if !isfile("data/casa_0.5-1.5keV.fits")
+        mkpath("data")
+        download("http://chandra.harvard.edu/photo/2009/casa/fits/casa_0.5-1.5keV.fits", "data/casa_0.5-1.5keV.fits")
+    end
+    if !isfile("data/casa_1.5-3.0keV.fits")
+        mkpath("data")
+        download("http://chandra.harvard.edu/photo/2009/casa/fits/casa_1.5-3.0keV.fits", "data/casa_1.5-3.0keV.fits")
+    end
+    if !isfile("data/casa_4.0-6.0keV.fits")
+        mkpath("data")
+        download("http://chandra.harvard.edu/photo/2009/casa/fits/casa_4.0-6.0keV.fits", "data/casa_4.0-6.0keV.fits")
+    end
+    if !isfile("data/ccd2rgd.jld")
```

Comment:
The file you're testing here is different from the file you're saving three lines below, so this test is always false and the files is always downloaded